views/profile_view/views.py

```diff
@@ -39,9 +39,12 @@ def get(self, request, username=None):
             return redirect('home')
         except Http404:
             return redirect('home')
-
         context = {
+            'request_username': f"{request.user.first_name} {request.user.last_name}",
```

Comment:
can we change this to full_name so it is easier to read

Reply:
Got it